app.js

```diff
@@ -130,12 +130,13 @@ app.use(
 app.use('/apply', applicationRouter);
 app.use('/', indexRouter);
 
-app.use((err, req, res, next) => {
-    if (err.code === 'EBADCSRFTOKEN') {
-        return res.status(403).render('503.njk');
-    }
-
-    return next(err);
+// Central error handler
+// https://www.joyent.com/node-js/production/design/errors
+// https://github.com/i0natan/nodebestpractices/blob/master/sections/errorhandling/centralizedhandling.md
```

Comment:
The central error handler is for things common to all errors, I'm not sure this is implemented correctly